gpcontrib/arenadata_toolkit/Makefile

```diff
@@ -5,8 +5,9 @@ MODULES = arenadata_toolkit
 EXTENSION = arenadata_toolkit
 DATA = arenadata_toolkit--1.0.sql
 
-REGRESS = arenadata_toolkit_test arenadata_toolkit_skew_test
+REGRESS = arenadata_toolkit_test arenadata_toolkit_skew_test adb_get_relfilenodes_test
 REGRESS_OPTS += --init-file=$(top_srcdir)/src/test/regress/init_file
+REGRESS_OPTS += --gpd_ignore_headers
```

Comment:
`make -C gpcontrib/arenadata_toolkit installcheck` seems broken:

```
make: Entering directory '/home/imber/Projects/arenadata/gpdb_src/gpcontrib/arenadata_toolkit'
make -C ../../src/test/regress pg_regress
make[1]: Entering directory '/home/imber/Projects/arenadata/gpdb_src/src/test/regress'
make -C ../../../src/port all
make[2]: Entering directory '/home/imber/Projects/arenadata/gpdb_src/src/port'
make -C ../backend submake-errcodes
make[3]: Entering directory '/home/imber/Projects/arenadata/gpdb_src/src/backend'
make[3]: Nothing to be done for 'submake-errcodes'.
make[3]: Leaving directory '/home/imber/Projects/arenadata/gpdb_src/src/backend'
make[2]: Leaving directory '/home/imber/Projects/arenadata/gpdb_src/src/port'
make -C ../../../src/common all
make[2]: Entering directory '/home/imber/Projects/arenadata/gpdb_src/src/common'
make -C ../backend submake-errcodes
make[3]: Entering directory '/home/imber/Projects/arenadata/gpdb_src/src/backend'
make[3]: Nothing to be done for 'submake-errcodes'.
make[3]: Leaving directory '/home/imber/Projects/arenadata/gpdb_src/src/backend'
make[2]: Leaving directory '/home/imber/Projects/arenadata/gpdb_src/src/common'
make[1]: Leaving directory '/home/imber/Projects/arenadata/gpdb_src/src/test/regress'
../../src/test/regress/pg_regress --inputdir=. --psqldir='/home/imber/builds/adb/3793/bin'    --init-file=../../src/test/regress/init_file --gpd_ignore_headers --dbname=contrib_regression arenadata_toolkit_test arenadata_toolkit_skew_test adb_get_relfilenodes_test
../../src/test/regress/pg_regress: unrecognized option '--gpd_ignore_headers'

Try "pg_regress -h" for more information.
make: *** [../../src/makefiles/pgxs.mk:303: installcheck] Error 2
make: Leaving directory '/home/imber/Projects/arenadata/gpdb_src/gpcontrib/arenadata_toolkit'
```

Also does this option needed after `regexp_replace` at tests?